libraries/RC_Channel/RC_Channel.h

```diff
@@ -178,6 +178,15 @@ class RC_Channel {
         KILL_IMU1 =          100, // disable first IMU (for IMU failure testing)
         KILL_IMU2 =          101, // disable second IMU (for IMU failure testing)
         CAM_MODE_TOGGLE =    102, // Momentary switch to cycle camera modes
+        KILL_ALL   =         103, // kill all motors
+        KILL_MOT1  =         104, // kill motor 1
+        KILL_MOT2  =         105, // kill motor 2
+        KILL_MOT3  =         106, // kill motor 3
+        KILL_MOT4  =         107, // kill motor 4
```

Comment:
Should I expand this to 8 motors? Anybody interested in that?

Reply:
Strictly it should yo up to 12, maybe better to have a bitmask param and single switch. For motors you can do it all with scripting, then we don't have the case where people set this up by mistake.

Reply:
perhaps you could reuse one of the plane only rc options so as to avoid burning one up just for testing/certification....like 87,88, or 89?